docs/extensions/tutorials/_posts/1970-01-01-PublishYourApp.md

```diff
@@ -7,48 +7,47 @@ section: Tutorials
 
 # Publish, Republish, Maintain
 
-You're satisfied how your app looks like and you want to **publish** it to the store. This tutorial shows you how to publish your app to both [App Store]({{ site.external.appleAppStore }}) and [Google Play]({{ site.external.googlePlayStore }}) store. We'll also describe what does it mean to republish and maintain the data in your app.
+You're satisfied with how your app looks and you want to **publish** it to the store. This tutorial shows you how to publish your app to both the [App Store]({{ site.external.appleAppStore }}) and [Google Play]({{ site.external.googlePlayStore }}). We'll also describe what it means to republish and how to maintain the data in your app.
 
 There are 2 ways you can publish your app:
+- using the Shoutem automated app publishing tool
```

Comment:
There was no link there, but I can add one if necessary, just let me know what to link.